Sources/BazelizeKit/Codegen/Codegen+Config.swift

```diff
@@ -13,11 +13,11 @@ import XCode
 
 /// https://bazel.build/docs/configurable-attributes
 extension Project {
-    // MARK: Public
+    // MARK: Internal
 
     /// build:debug --//:mode=debug
     /// bazel build --config=debug [PACKAGE:RULE]
-    public func generateBazelRC(_ kit: Kit) throws {
+    internal func generateBazelRC(_ kit: Kit) throws {
```

Comment:
This `internal` is used by default. Is it added by the linter?

Reply:
no

Reply:
if it was not added by the linter, could we remove `internal`?

Reply:
ok